app/components/ComparisonTable/ComparisonTable.njk

```diff
@@ -0,0 +1,111 @@
+{% macro ComparisonTable() %}
+  <div  class="govuk-width-container" role="region" aria-labelledby="table-caption" tabindex="0">
+    <table class="govuk-table responsive-table-four responsive-table-four-v3">
+      <caption id="table-caption" class="govuk-table__caption govuk-visually-hidden">Dates and amounts</caption>
```

Comment:
Still visible. I think the class name needs to change to `visually-hidden`?
Not sure about the others.